metricbeat/.gitignore

```diff
@@ -3,3 +3,4 @@ build
 /metricbeat
 /metricbeat.test
 /docs/html_docs
+/logs/*
```

Comment:
This should be not needed anymore as soon as this is rebased on top of master or master is merged.